src/arc.md

```diff
@@ -0,0 +1,13 @@
+# Implementing Arc
+
+In this section, we'll be implementing a simpler version of `std::sync::Arc`.
+Similarly to [the implementation of `Vec` we made earlier](vec.md), we won't be
+taking advantage of as many optimizations, intrinsics, or unstable code as the
+standard library may.
+
+This implementation is loosely based on the standard library's implementation
+(technically taken from `alloc::sync` in the latest nightly), but it will not
```

Comment:
The latest nightly will desync from this version eventually. You should probably give a date to this nightly and link to the file in the commit of the nightly you are talking about.

Reply:
I'll just make sure that nightly is in sync with stable right now and just use stable 1.49 as a base